Lib/test/test_asyncio/test_queues.py

```diff
@@ -169,12 +169,14 @@ def test_nonblocking_get_exception(self):
 
     async def test_get_cancelled(self):
         q = asyncio.Queue()
+        started = asyncio.Event()
 
         async def queue_get():
+            started.set()
             return await asyncio.wait_for(q.get(), 0.051)
```

Comment:
is the wait_for() call still useful?

Reply:
The whole test is redundant IMHO.
Please take a look on https://github.com/python/cpython/pull/32040 for alternative fix that affects more problematic tests.